ufw

```diff
@@ -0,0 +1,29 @@
+# To check the status of the firewall:
+ufw status
+
+# To check the status with more details:
+ufw status verbose
+
+# To allow a port on the firewall by its name:
+ufw allow <port-name>
```

Comment:
```suggestion
ufw allow <PORT-NAME>
```

Optional fix: missing placeholder capitalization [change placeholder naming style of the first placeholders or change the current placeholder naming style].

Reply:
Thank you again